main/guides/getting-started/Explainer - Deploying a Smart Contact.md

```diff
@@ -0,0 +1,22 @@
+# Deploying a Smart Contact
+- To deploy your Agoric Smart Contract, follow the simple steps outlined in this explainer.
+- As you're going through this explainer it may be helpful to watch this video walkthrough, showing you all the steps and the desired outcome of each.
+- TODO: INSERT VIDEO
+
+## Prerequisities:
+- Deploying a Smart Contract requires the Agoric SDK. For steps on installing the Agoric SDK see the Explainer on [Starting a Local Chain](https://github.com/Agoric/documentation/blob/kbennett2000-patch-1/main/guides/getting-started/Explainer%20-%20How%20to%20Start%20a%20Local%20Chain.md)
+
+## Create a Dapp
+- Use the `agoric init` command to initialize a new project.
+- Example: 
+```sh
+agoric init my-first-dapp
```

Comment:
[agoric init](https://docs.agoric.com/guides/agoric-cli/#agoric-init) is deprecated in favor of `yarn create @agoric/dapp` as in getting started.

Reply:
fixed